### PR TITLE
Create qg organisations and sites

### LINKED ISSUE #341: Additions to GSQ Alias
For DES to use the Catalogue, we need to create a list of our organisation and org units. These critically link to publishers in the metadata schema.

@DavidCrosswellGSQ @KellyVance I'm proposing to add items and refine the [GSQ Alias](https://github.com/geological-survey-of-queensland/vocabularies/blob/master/vocabularies/gsq-alias.ttl) file to  include this information and build out some of the ORG ontology items.
cc @UbuntuScott

### PR DESCRIPTION
**Pull request by broughd-qld**
This is a starting point list of the agents (organisations / parties / agencies) and sites for the DES Data Catalogue. Related conversations in [Issue 341](https://github.com/geological-survey-of-queensland/vocabularies/issues/341)

This will be separate to the GSQ Parties database, but will provide a high level list of agents and their changes.

@johnmckellar including you as FYI based on advice from @KellyVance.
RESOLVES #341

### CODE REVIEW

**Comment by broughd-qld** [comment]
Have fixed up the typos and it now ready for review.

File: vocabularies/qg-agents.ttl [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+<https://linked.data.gov.au/qg-sites>  rdf:type owl:Ontology, skos:ConceptScheme ;
+        owl:imports <http://www.w3.org/2006/vcard/ns> ,
+            org: ,
+            <http://www.w3.org/ns/prov-o#> ;
+        dcterms:created "2020-05-21" ;
+        dcterms:creator <https://orcid.org/0000-0002-3884-3420> ,
+                        <https://orcid.org/0000-0003-0395-9703> ,
+                        <http://linked.data.gov.au/org/des> ;
+        dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+        dcterms:modified "2020-11-27" ;
```
Comment (UbuntuScott) [inline vocabularies/qg-agents.ttl:111]: `   dcterms:modified "2020-12-01" ;`

File: vocabularies/qg-agents.ttl [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+<https://linked.data.gov.au/qg-agent> rdf:type owl:Ontology , skos:ConceptScheme ;
+      owl:imports <http://www.w3.org/2006/vcard/ns> ,
+         org: ,
+         <http://www.w3.org/ns/prov-o#> ;
+      dcterms:created "2020-05-21" ;
+      dcterms:creator <https://orcid.org/0000-0002-3884-3420> ,
+                      <https://orcid.org/0000-0003-0395-9703> ,
+                      <http://linked.data.gov.au/org/des> ;
+      dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+      dcterms:modified "2020-11-27" ;
```
Comment (UbuntuScott) [inline vocabularies/qg-agents.ttl:27]:
```suggestion
      dcterms:modified "2020-12-01" ;

```

File: vocabularies/qg-agents.ttl (hunk 1 above)
Comment (UbuntuScott) [inline vocabularies/qg-agents.ttl:111]:
```suggestion
        dcterms:modified "2020-12-01" ;

```

**Review by UbuntuScott — CHANGES_REQUESTED** [review]
Looks good @broughd-qld 
A combination of qgorg and qgsite. 
two things in my review : 
1. Just the update dates - request change to today's date
2. Question:  Do we need to identify a skos:broader org for the leaf sites at the end?  I'm not sure why JCU is there, is that one of our sites? 
Or are they top concepts? or something else?

**Comment by broughd-qld** [comment]
@UbuntuScott the JCU site is the location of a satellite office / work team.

**Review by UbuntuScott — APPROVED** [review]

**Review by UbuntuScott — APPROVED** [review]

**Comment by broughd-qld** [comment]
@DavidCrosswellGSQ I have split the agents (organisations) from the sites (locations) in the file as per our discussion. 2 files now ready for review.

File: vocabularies/qg-sites.ttl [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                        <https://orcid.org/0000-0003-0395-9703> ,
+                        <http://linked.data.gov.au/org/des> ;
+        dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+        dcterms:modified "2020-12-01" ;
+        dcterms:title "Queensland Government office locations"@en ;
+        skos:prefLabel "QG Offices"@en ;
+        rdfs:comment "RDF graph of Queensland Government locations, used as an initial starting point for the DES Data Catalogue. The selected office locations may be used as part of contact details for the catalogue, not all locations have public access or information facilities."@en .
+
+
+###  https://linked.data.gov.au/qg-sites/Ecosciences
```
[the remark this answers is not in the document]
Comment (DavidCrosswellGSQ) [reply vocabularies/qg-sites.ttl:33]: Please remove this comment throughout the vocab.

File: vocabularies/qg-sites.ttl [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+qgsite:Ecosciences rdf:type owl:NamedIndividual ,
```
[the remark this answers is not in the document]
Comment (DavidCrosswellGSQ) [reply vocabularies/qg-sites.ttl:34]: Please use the SKOS "a" throughout the document.

File: vocabularies/qg-agents.ttl [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+ 
+@base <https://linked.data.gov.au/qg-agent> .
+
+<https://linked.data.gov.au/qg-agent> rdf:type owl:Ontology , skos:ConceptScheme ;
```
[the remark this answers is not in the document]
Comment (DavidCrosswellGSQ) [reply vocabularies/qg-agents.ttl:18]: Please replace rdf:type with the SKOS "a" throughout this document.

File: vocabularies/qg-agents.ttl [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+###  https://linked.data.gov.au/qg-agent/DES
+qgorg:DES rdf:type owl:NamedIndividual ,
+              org:FormalOrganization ,
+              prov:Organization ,
+              skos:Concept ;
+     org:hasPrimarySite qgsite:GeorgeSt ;
+     org:hasSite qgsite:Ecosciences ;
+     rdfs:label "Department of Environment and Science"@en ;
+     skos:prefLabel "Department of Environment and Science"@en ;
+     skos:altLabel "DES"@en ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/qg-agents.ttl:42]: Is DES notation?

File: vocabularies/qg-agents.ttl [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+
+###  https://linked.data.gov.au/qg-agent/DES
+qgorg:DES rdf:type owl:NamedIndividual ,
+              org:FormalOrganization ,
+              prov:Organization ,
+              skos:Concept ;
+     org:hasPrimarySite qgsite:GeorgeSt ;
+     org:hasSite qgsite:Ecosciences ;
+     rdfs:label "Department of Environment and Science"@en ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/qg-agents.ttl:40]:
```suggestion

```

Comment (DavidCrosswellGSQ) [reply vocabularies/qg-agents.ttl:40]: Please remove rdfs:label throughout the document. It is duplicative of SKOS:preflabel.

File: vocabularies/qg-agents.ttl [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+                      <https://orcid.org/0000-0003-0395-9703> ,
+                      <http://linked.data.gov.au/org/des> ;
+      dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+      dcterms:modified "2020-12-01" ;
+      dcterms:title "Queensland Governement organizations and agencies"@en ;
+      skos:prefLabel "QG Organisations"@en ;
+      rdfs:comment "RDF graph of Queensland Government organisations (departments) which may include some of the departmental divisions, used as an initial starting point for the DES Data Catalogue. Not a full organisational hierarchy but limited to higher levels which are useful to identify."@en .
+
+
+###  https://linked.data.gov.au/qg-agent/DES
```
Comment (DavidCrosswellGSQ) [inline vocabularies/qg-agents.ttl:33]:
```suggestion

```

Comment (DavidCrosswellGSQ) [reply vocabularies/qg-agents.ttl:33]: Please remove these comments throughout the document

**Review by DavidCrosswellGSQ — CHANGES_REQUESTED** [review]

File: vocabularies/qg-agents.ttl [hunk 9]
```diff
@@ -0,0 +1,102 @@
+@prefix qgorg: <https://linked.data.gov.au/qg-agent/> .
+@prefix qgsite: <https://linked.data.gov.au/qg-sites/> .
```
[the remark this answers is not in the document]
Comment (broughd-qld) [reply vocabularies/qg-agents.ttl:3]: this vocabulary is referred into the file. Refer to line 38

File: vocabularies/qg-agents.ttl (hunk 6 above)
Comment (broughd-qld) [reply vocabularies/qg-agents.ttl:38]: I was thinking more altLabel, we could go to notation. My only concern is that notation is unique with a scheme and there could be different names using DES in the future. 
Or we just take a chance and use notation now.

File: vocabularies/qg-agents.ttl [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+         org: ,
+         <http://www.w3.org/ns/prov-o#> ;
+      dcterms:created "2020-05-21" ;
+      dcterms:creator <http://linked.data.gov.au/org/des> ;
+      dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+      dcterms:modified "2020-12-01" ;
+      dcterms:title "Queensland Government organisations and agencies"@en ;
+      skos:prefLabel "QG Organisations"@en ;
+      rdfs:comment "RDF graph of Queensland Government organisations (departments) which may include some of the departmental divisions, used as an initial starting point for the DES Data Catalogue. Not a full organisational hierarchy but limited to higher levels which are useful to identify."@en .
+
```
Comment (KellyVance) [inline vocabularies/qg-agents.ttl:30]:
```suggestion

<http://linked.data.gov.au/org/des>  
    a sdo:Organization ;
    sdo:name "Department of Environment and Science"@en .

<http://linked.data.gov.au/org/gsq> a sdo:Organization ;
    sdo:name "Geological Survey of Queensland"@en ;
    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
    
```

File: vocabularies/qg-sites.ttl [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+        owl:imports <http://www.w3.org/2006/vcard/ns> ,
+            org: ,
+            <http://www.w3.org/ns/prov-o#> ;
+        dcterms:created "2020-05-21" ;
+        dcterms:creator <http://linked.data.gov.au/org/des> ;
+        dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+        dcterms:modified "2020-12-01" ;
+        skos:prefLabel "Queensland Government Offices"@en ;
+        rdfs:comment "RDF graph of Queensland Government locations, used as an initial starting point for the DES Data Catalogue. The selected office locations may be used as part of contact details for the catalogue, not all locations have public access or information facilities."@en .
+
```
Comment (KellyVance) [inline vocabularies/qg-sites.ttl:25]:
```suggestion
<http://linked.data.gov.au/org/des>  
    a sdo:Organization ;
    sdo:name "Department of Environment and Science"@en .

<http://linked.data.gov.au/org/gsq> a sdo:Organization ;
    sdo:name "Geological Survey of Queensland"@en ;
    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
```

**Review by UbuntuScott — APPROVED** [review]

**Pull request merged**